### DISCUSSION TITLE
Add document how to use configuration file

### DISCUSSION THREAD

**Discussion by ghost** [actix/actix-web#2697] [Ideas] [answered]
Configuration file is most common part of app development. 
We using configure file storage database connection ,password, email send settings. API keys and secret.

How to put them together with actix-web app?

what's the best practice?

**Comment by robjtede** [comment, marked as answer]
This is a more general application development question, Actix Web doesn't provide any configuration file functionality because it is out of scope. As such, we have no reccommendation of best practice. Though, we'd be happy to accept a submission to the examples repo for a more fully-fledged configuration example project.
